src/lib/journey/callbacks/unknown/unknown.svelte

```diff
@@ -0,0 +1,9 @@
+<script lang="ts">
+  import type { FRCallback } from '@forgerock/javascript-sdk';
+
+  export let callback: FRCallback;
+
+  const type = callback.getType();
+</script>
+
+<p>Unknown callback encountered: {type}. Please contact support.</p>
```

Comment:
We probably want to do something better than this, but I needed a fallback component for the `mapCallbackToComponent` function.

Reply:
Thinking about it, perhaps we might want to add something to the journey object that we expose, to show that an unknown callback was encountered?

Reply:
By "journey" object, you mean the object provided to the application upon the `onSuccess` event? If so, that's in interesting idea, but I'd have to really think about how to do that. The way this code works is it's very unidirectional. Data flows from parent to child, but not child to parent. This is intentional and goes back to the Flux architectural pattern from Facebook's React team.

Being that this is deeper in the app than the store that is returned to the app, the communication would be against the natural flow of the code. But, that doesn't mean we couldn't figure out a way to do such a thing.